docs/metadata/prophecy-build-tool.md

```diff
@@ -96,12 +107,35 @@ Querying existing jobs to find current job: Offset: 0, Pagesize: 25
 ✅ Deployment completed successfully!
 ```
 
+The `deploy` command also supports an advanced option `--dependent-projects-path` if there is a need to build projects other than the main project that has to be deployed.
+This would be useful if there are dependent Pipelines whose source code can be cloned into a different directory accessible to PBT
+while running `deploy` for the main project.
+
+```shell
+pbt deploy --help
+Prophecy-build-tool v1.0.3.3
+
+Usage: pbt deploy [OPTIONS]
+
+Options:
+  --path TEXT                     Path to the directory containing the
+                                  pbt_project.yml file  [required]
+  --dependent-projects-path TEXT  Dependent projects path
+  --release-version TEXT          Release version to be used during
+                                  deployments
+  --project-id TEXT               Project Id placeholder to be used during
+                                  deployments
+  --prophecy-url TEXT             Prophecy URL placeholder to be used during
+                                  deployments
+  --help                          Show this message and exit.
+```
+
 #### Running all unit tests in project
 
-Running unit tests requires **FABRIC_NAME** environment variable to be set. This will be used to pick the correct configuration for running the unit tests. Example:
+PBT supports running unit tests inside the Prophecy project. Unit tests run with the `default` configuration unless explicitly overridden with the `FABRIC_NAME` environment variable
```

Comment:
What exactly is `FABRIC_NAME`?

Reply:
Removed FABRIC_NAME entirely from the document as its not needed anymore. Even in prophecy UI unit tests only run with the default config